Assets/Scripts/Characters/Human/Hero.cs

```diff
@@ -181,6 +181,8 @@ public class Hero : Human
     private float uTapTime = -1f;
     private bool wallJump;
     private float wallRunTime;
+    private bool ReelInInput=false;
```

Comment:
The default value of a bool is false. Thereby it's not required to state that they are false.